Tests/Microsoft.AppCenter.Test.Functional/Distribute/DistributeUpdateTest.cs

```diff
@@ -197,23 +191,27 @@ public async Task CheckForUpdateTest(UpdateTrack updateTrack, string urlDiff)
             await Distribute.IsEnabledAsync();
 
             // Wait for processing event.
-            var result = await implicitCheckForUpdateTask;
+            var resultImplicit = await implicitCheckForUpdateTask;
 
             // Verify response.
-            Assert.Equal("GET", result.Method);
-            Assert.True(result.Uri.Contains(urlDiff));
-            Assert.True(result.Uri.Contains(Config.ResolveAppSecret()));
+            Assert.Equal("GET", resultImplicit.Method);
+            Assert.Contains(urlDiff, resultImplicit.Uri);
+            Assert.Contains(Config.ResolveAppSecret(), resultImplicit.Uri);
+
+            // Wait a 5s for give time to complete internal processes.
+            await Task.Delay(5000);
```

Comment:
This should work, but I don't think this fix is correct

Reply:
This is the initial implementation we had (we had this and something like `Task.WhenAny`...).

Reply:
This way to avoid failing tests, because some internal operations need time to complete.
In failing cases, I found this log that appears after call `CheckForUpdate`:
`A check for update is already ongoing.`
And I suspect that [this](https://github.com/microsoft/appcenter-sdk-android/blob/develop/sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/Distribute.java#L499) returns false because `mCheckReleaseCallId` is still not null.

Reply:
I see. Ok, resolving this PR. Thanks for explanation o/